tests/jax/test_custom_call_compute.py

```diff
@@ -24,8 +24,13 @@
 from transformer_engine.jax.layernorm import layernorm
 from transformer_engine.jax.mlp import fp8_ln_mlp
 
-GEMM_CASES = [(256, 256, 512), (32, 32, 32), (16384, 1024, 2816), (16384, 2816, 1024),
-              (16384, 1024, 1024)]
+GEMM_CASES = [
+    (256, 256, 512),
+    (32, 32, 32),
+    (2048, 1024, 2048),
```

Comment:
Is there a reason to make these matrices smaller?

Reply:
https://github.com/NVIDIA/TransformerEngine/pull/397#issuecomment-1692492335

The tests with integer matrices failed with the larger sizes when I ran on an L40. cuBLAS picked GEMM kernels that were not quite accurate enough to get bit-wise exact results, I think because it did split-k GEMM. I just reduced the matrix sizes to get back to bit-wise exact results, but we could alternatively modify the integer matrix tests with FP8 numerical tolerances.